### DISCUSSION TITLE
How to access everyones' conversation log through DIFY API ?

### DISCUSSION THREAD

**Discussion by kdasfij** [langgenius/dify#10048] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

How to get everyones' LOG LIST include "User" & "Conversation_id"  through DIFY API ?

**Comment by crazywoola** [comment]
Can you describe why you need this?
